Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/NotificationUtils.java

```diff
@@ -95,6 +86,7 @@ public class NotificationUtils extends AppCompatActivity {
     private static boolean hasStartedService = false;
     public static OverlaySheetService.OverlayBinder binder;
     public static ArrayList<Bundle> listData = new ArrayList();
+    public static ArrayList<String> removeList = new ArrayList<>();
```

Comment:
can we not rely on static variable for this if values are getting restored after killing and opening the app